src/app/public/modules/infinite-scroll/infinite-scroll-dom-adapter.service.ts

```diff
@@ -54,24 +54,18 @@ export class SkyInfiniteScrollDomAdapterService implements OnDestroy {
    * is visible (or scrolled to) within its scrollable parent container.
    * @param elementRef The infinite scroll element reference.
    */
-  public scrollTo(elementRef: ElementRef): Observable<void> {
+  public scrollTo(elementRef: ElementRef): Observable<any> {
```

Comment:
Is this an `Observable<boolean>`?

Reply:
Also, it doesn't look like you're using the return value of this observable in the consuming class?

Reply:
I'm not using the value that this observable returns, I only care that the event happened. I believe it returns an event object, but I would rather this stay as a `void` return type since I have no intention of using the value. I can't set it to void though since it does actually have a value.

Reply:
Got it back to void return type with a map